### PR TITLE
Add home page

### PR DESCRIPTION
**Pull request by MrTimscampi**
image

### CODE REVIEW

File: src/components/HomepageFeatures.js [hunk 1]
```diff
@@ -4,40 +4,73 @@ import styles from './HomepageFeatures.module.css';
...
+        Listen to music, your artists and your playlists, at home or on the go.
       </>
     ),
   },
+  {
+    title: 'Live TV & DVR',
+    Svg: require('../../static/img/undraw_game_day.svg').default,
+    description: (
+      <>
+        Watch TV and set automatic recordings to expand your library.
```
Comment (Artiume) [inline src/components/HomepageFeatures.js:38]: should we mention DVR for the method of recording?

File: src/components/HomepageFeatures.js [hunk 2]
```diff
@@ -4,40 +4,73 @@ import styles from './HomepageFeatures.module.css';
...
+      </>
+    ),
+  },
+  {
+    title: 'Books',
+    Svg: require('../../static/img/undraw_book_lover.svg').default,
+    description: (
+      <>
+        Store, manage and read your books, comics and magazines.
```
Comment (Artiume) [inline src/components/HomepageFeatures.js:47]: oxford comma? Also, the double 'and' makes the sentence a little awkward

File: src/components/HomepageFeatures.js [hunk 3]
```diff
@@ -4,40 +4,73 @@ import styles from './HomepageFeatures.module.css';
...
+        Organize your photos and share memories with your friends and family.
+      </>
+    ),
+  },
+  {
+    title: 'SyncPlay',
+    Svg: require('../../static/img/undraw_real_time_collaboration.svg').default,
+    description: (
+      <>
+        Sharing a movie night with has never been so easy.
```
Comment (Artiume) [inline src/components/HomepageFeatures.js:65]: friends

File: src/components/MoreClients.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Web from '../../static/img/web.svg';
+import Desktop from '../../static/img/monitor.svg';
+import { Android, Apple, Roku, Amazon, Kodi } from '@icons-pack/react-simple-icons';
```
Comment (Artiume) [inline src/components/MoreClients.js:4]: is there a chromecast icon?

Comment (MrTimscampi) [reply src/components/MoreClients.js:4]: There is, but that's not really a client per se (As in you can't run it by itself, it needs another client).

I see it more as a feature than as a client, hence it not being included here.

File: src/components/HomepageFeatures.js (hunk 1 above)
Comment (MrTimscampi) [reply src/components/HomepageFeatures.js:38]: Probably not, it already mentioned in the line right above: "Live TV & DVR".

**Pull request merged**